here/traffic/here_api.py

```diff
@@ -47,6 +47,20 @@ def get_access_token(key_id, key_secret, token_url):
     LOGGER.info('Getting Access Token')
     r = requests.post(token_url, auth=oauth1, json=payload, headers=headers)
 
+    try:
+        r.raise_for_status()
+    except KeyError as k_err:
+        LOGGER.error(k_err)
+    except requests.exceptions.HTTPError as err:
+        LOGGER.error('Error in requesting access token')
+        LOGGER.error(err)
+        try:
+            err_msg = query_response.json()['message']
+        except JSONDecodeError:
+            err_msg = query_response.text
+        finally:
+            raise HereAPIException(err_msg)
+
     access_token = r.json()['accessToken']
```

Comment:
this line should be in the try block

Reply:
should it be catching different errors than line 50 to 62? Just Jsondecodeerror? and maybe another keyerror?